challenge-06/challenge-06.js

```diff
@@ -58,9 +70,40 @@ Crie uma função chamada `convertToHex`, com as seguintes características:
     a frase:
     "Não temos o equivalente hexadecimal para [COR]."
 */
-// ?
-
+function convertToHex(color) {
+  var pink = '#cc33aa';
+  var yellow = '#e3ff00';
+  var red = '#fe0a0a';
+  var green = '#05ff00';
+  var blue = '#0500ff';
+  switch (color) {
+    case 'pink':
+      console.log('O hexadecimal para a cor ' + color + ' é ' + pink + '.');
```

Comment:
-1
Dentro de funções, sempre use o `return` para devolver valores. O `console.log` deve ser usado somente para mostrar os valores que as funções irão retornar, ok?

Outro detalhe aqui: ao invés de setar todas as cores já de início, crie uma variável que será alterada conforme a cor escolhida, e retorne a frase somente no final da função. Faça a verificação da cor dentro do switch e mude o valor da variável conforme o parâmetro passado :)

O código vai ficar muito mais conciso :)